config/testnet/docker-compose.yml

```diff
@@ -5,26 +5,23 @@ services:
   storage01:
     image: nspccdev/neofs-storage-testnet:0.21.1
     container_name: neofs-testnet
+    env_file: node_config.env
     network_mode: host
     restart: always
     volumes:
-      - neofs_storage:/storage
+      - <NEOFS_STORAGE>:/storage
```

Comment:
In README.md we have

```
NeoFS objects will be stored on your machine. By default, docker-compose 
configured to store objects in named docker volume `neofs_storage`. You can 
specify directory on the filesystem to store objects there.
```

Can we keep object storage in a docker volume by default?

Reply:
agree, fixed